### ISSUE TITLE
Drop support for native-to-standard space transforms

### ISSUE THREAD

**Issue by tsalo** [ME-ICA/aroma#11]
### Summary
This stems from https://github.com/Brainhack-Donostia/ica-aroma-org/pull/41#issuecomment-723421163. We will drop support for native-to-standard space transforms for now, and all native-space data must be accompanied with tissue probability maps for now.

### Next Steps
1. Remove all transforms.
2. Drop FEAT directory support too, maybe?

**Comment by eurunuela** [comment]
Okay, so shall we drop all these dependencies and use probability maps until [poldracklab/nitransforms#51](https://github.com/poldracklab/nitransforms/pull/51) is merged?

**Comment by oesteban** [comment]
I think that was @tsalo's original proposal - do not bother at this point about how the TPMs are generated in the native space, just assume they are defined there.

**Comment by eurunuela** [comment]
> I think that was @tsalo's original proposal - do not bother at this point about how the TPMs are generated in the native space, just assume they are defined there.

Yes. I wanted to make sure. I was starting to get lost in the conversation having now two different repos after the Brainhack Donostia tutorial 😅
